controllers/limited_support/limited_support_controller_test.go

```diff
@@ -72,19 +76,22 @@ limited_support_enabled{name="osd_exporter"} 1
 	}
 
 	for _, tc := range []struct {
-		name            string
-		expectedResults string
+		name             string
+		expectedResults  string
+		testEnvClusterID string
 	}{
 		{
-			name: "limited-support invalid configMap",
+			name:             "limited-support invalid configMap",
+			testEnvClusterID: "i-am-a-cluster-id",
 			expectedResults: `
 # HELP limited_support_enabled Indicates if limited support is enabled
 # TYPE limited_support_enabled gauge
-limited_support_enabled{name="osd_exporter"} 0
+limited_support_enabled{_id="i-am-a-cluster-id",name="osd_exporter"} 0
 `,
 		},
 	} {
 		t.Run(tc.name, func(t *testing.T) {
+			os.Setenv(EnvClusterID, tc.testEnvClusterID)
```

Comment:
Same idea here :D Super cool the tests pass now!

```suggestion
			t.Setenv(EnvClusterID, tc.testEnvClusterID)
```